teaching_reproducibility.Rmd

```diff
@@ -92,17 +139,21 @@ are not very clear to learners about where the error is coming from
 and what should be done next to fix the problem. 
 
 ```
-# Add a figure here that illustrates an example of an R Markdown doc being  
+# TODO Add a figure here that illustrates an example of an R Markdown doc being  
```

Comment:
Possible candidate screenshot, a latex error would be better though since that is talked about in the para above.

![image](https://user-images.githubusercontent.com/4560057/127529483-ccc41870-d293-492a-bf40-d459f1e07521.png)

Reply:
Here's a latex one I could dig up (derived from Overleaf docs: https://www.overleaf.com/learn/latex/Errors/Missing_$_inserted)

<img width="1111" alt="Screen Shot 2021-07-29 at 6 00 48 PM" src="https://user-images.githubusercontent.com/6865016/127584123-f3aef254-6ca8-423a-9bc8-d1d8c36d0f06.png">

Reply:
Added!